### PR TITLE
n26631 add IRC log

### PR DESCRIPTION
**Pull request by LarryRuane**

### CODE REVIEW

File: _posts/2022-12-14-#26631.md [hunk 1]
```diff
@@ -122,9 +122,273 @@ is just its physical size, but an input typically
    Bonus: is there a way to create such an output script where this unspendability
    can even be mathematically proven?
 
- <!-- TODO: After meeting, uncomment and add meeting log between the irc tags
  ## Meeting Log
```
Comment (stickies-v) [inline _posts/2022-12-14-#26631.md:125]: Looks like the leading whitespace is messing up with md formatting:
image

```suggestion
## Meeting Log
```

File: _posts/2022-12-14-#26631.md [hunk 2]
```diff
@@ -122,9 +122,273 @@ is just its physical size, but an input typically
...
+17:02 <LarryRuane> #startmeeting
+17:00 <d33r_gee> hello
+17:00 <stickies-v> hi
+17:01 <rozehnal_paul> hi
+17:01 <Jmy> Hi everyone
+17:01 <andrewtoth_> hi
+17:01 <guest647> hi
+17:01 <b_101_> hi
+17:01 <ishaanam[m]> hi
+17:02 <LarryRuane> hi!
```
Comment (stickies-v) [inline _posts/2022-12-14-#26631.md:137]: I'd reorder the #startmeeting line back to its actual position:
```suggestion
17:00 <d33r_gee> hello
17:00 <stickies-v> hi
17:01 <rozehnal_paul> hi
17:01 <Jmy> Hi everyone
17:01 <andrewtoth_> hi
17:01 <guest647> hi
17:01 <b_101_> hi
17:01 <ishaanam[m]> hi
17:02 <LarryRuane> #startmeeting
17:02 <LarryRuane> hi!
```

**Review by stickies-v — COMMENTED** [review]
Thanks @LarryRuane ! Some formatting comments, and also status still has to be updated to `past`

_edit: fixed it already_

**Comment by glozow** [comment]
Thanks @LarryRuane and @stickies-v!

**Pull request merged**